spark-bigtable-core/src/main/scala/com/google/cloud/spark/bigtable/datasources/BigtableSparkConf.scala

```diff
@@ -69,6 +69,9 @@ object BigtableSparkConf {
   @ObsoleteApi("This field is obsolete is will be removed on a future version. To use a default configuration simply leave it unset")
   val DEFAULT_BIGTABLE_PUSH_DOWN_FILTERS = true
 
+  /** The maximum number of cell versions to return per column. */
+  val BIGTABLE_MAX_VERSIONS: String = SparkScanConfig.MAX_VERSIONS_CONFIG_KEY
```

Comment:
Sorry for missing this in the previous comment, can we also update here to be BIGTABLE_READ_MAX_VERSIONS? thanks!